### PR TITLE
RELATED: PSDK-173 add support for user management APIs

### PR DESCRIPTION
**Pull request by hkad98**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: docker-compose.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ version: '3.7'
 services:
   gooddata-cn-ce:
     # Use the dev_latest tag which points to the latest development
-    image: registry.gitlab.com/gooddata/gdc-nas/gooddata-cn-ce:stable
+    image: 020413372491.dkr.ecr.us-east-1.amazonaws.com/nas/gooddata-cn-ce:stable
```
Comment (jaceksan) [inline docker-compose.yaml:10]: The comment above talks about the latest tag, but stable tag is used.
Please, explain shortly why we use stable.

**Review by jaceksan — CHANGES_REQUESTED** [review]

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by jaceksan — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR589

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/336/ : FAILURE in 1m 05s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/486/ : SUCCESS in 53s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/490/ : SUCCESS in 6s

**Pull request merged**